<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 61, 85);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>209</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="radio_holdwait">
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	font: 14pt &quot;Futura Bk BT&quot;;
	color:  rgb(217, 217, 217);
    spacing: 20px;
	background-color: none;
}

QRadioButton::hover {
	color: rgb(105, 111, 128);
}

QRadioButton::indicator::hover {
	color: rgb(105, 111, 128);
}

QRadioButton::indicator {
    width: 19px;
    height: 19px;
}

QRadioButton::indicator:unchecked {
	
	image: url(:/resources/checkBox_off.png);
}

QRadioButton::indicator:checked {
	
	image: url(:/resources/checkBox_on.png);
}</string>
      </property>
      <property name="text">
       <string>Hold and Wait</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_preempt">
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	font: 14pt &quot;Futura Bk BT&quot;;
	color:  rgb(217, 217, 217);
    spacing: 20px;
background-color: none;
}

QRadioButton::indicator {
    width: 19px;
    height: 19px;
}

QRadioButton::indicator:unchecked {
	
	image: url(:/resources/checkBox_off.png);
}

QRadioButton::indicator:checked {
	
	image: url(:/resources/checkBox_on.png);
}
</string>
      </property>
      <property name="text">
       <string>No Preemption</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_circularw">
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	font: 14pt &quot;Futura Bk BT&quot;;
	color:  rgb(217, 217, 217);
    spacing: 20px;
background-color: none;
}

QRadioButton::indicator {
    width: 19px;
    height: 19px;
}

QRadioButton::indicator:unchecked {
	
	image: url(:/resources/checkBox_off.png);
}

QRadioButton::indicator:checked {
	
	image: url(:/resources/checkBox_on.png);
}</string>
      </property>
      <property name="text">
       <string>Circular Wait</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_distribution">
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	font: 14pt &quot;Futura Bk BT&quot;;
	color:  rgb(217, 217, 217);
    spacing: 20px;
background-color: none;
}

QRadioButton::indicator {
    width: 19px;
    height: 19px;
}

QRadioButton::indicator:unchecked {
	
	image: url(:/resources/checkBox_off.png);
}

QRadioButton::indicator:checked {
	
	image: url(:/resources/checkBox_on.png);
}</string>
      </property>
      <property name="text">
       <string>Carefull Distribution</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_bankier">
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	font: 14pt &quot;Futura Bk BT&quot;;
	color:  rgb(217, 217, 217);
    spacing: 20px;
background-color: none;
}

QRadioButton::indicator {
    width: 19px;
    height: 19px;
}

QRadioButton::indicator:unchecked {
	
	image: url(:/resources/checkBox_off.png);
}

QRadioButton::indicator:checked {
	
	image: url(:/resources/checkBox_on.png);
}</string>
      </property>
      <property name="text">
       <string>Bankier-Algorithm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="startSimulationButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 5px;
	background-color: rgb(105, 111, 128);
	font: 18pt &quot;Futura BdCn BT&quot;;
	color: rgb(217, 217, 217);
};
QPushButton::pressed {
    border-radius: 5px;
	background-color: rgb(40, 43, 52);
	font: 18pt &quot;Futura BdCn BT&quot;;
	color: rgb(217, 217, 217);
}
</string>
   </property>
   <property name="text">
    <string>Start Simulation</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 5px;
	background-color: rgb(105, 111, 128);
	font: 18pt &quot;Futura BdCn BT&quot;;
	color: rgb(217, 217, 217);
};
QPushButton::pressed {
    border-radius: 5px;
	background-color: rgb(40, 43, 52);
	font: 18pt &quot;Futura BdCn BT&quot;;
	color: rgb(217, 217, 217);
}
</string>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Printer">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>343</y>
     <width>49</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    padding-right: 15px; /* make room for the arrows */
    border-width: 5px;
	border-radius: 3px;
	background-color: rgb(105, 111, 128);
    padding: 8px;
}

QSpinBox::up-button {
    subcontrol-origin: border;
	border-radius: 3px;
    subcontrol-position:  top right; /* position at the top right corner */
	background-color: rgb(152, 161, 186);
    width: 20px; /* 16 + 2*1px border-width = 15px padding + 3px parent border */
    border-width: 3px;
}

QSpinBox::up-button:hover {
}

QSpinBox::up-button:pressed {
}

QSpinBox::up-arrow {
    width: 7px;
    height: 7px;
}

QSpinBox::up-arrow:disabled, QSpinBox::up-arrow:off { /* off state when value is max */

}

QSpinBox::down-button {
      subcontrol-origin: border;
	border-radius: 3px;
    subcontrol-position:  bottom right; /* position at the top right corner */
	background-color: rgb(152, 161, 186);
    width: 20px; /* 16 + 2*1px border-width = 15px padding + 3px parent border */
    border-width: 3px;
}

QSpinBox::down-button:hover {
    border-image: url(:/images/spindown_hover.png) 1;
}

QSpinBox::down-button:pressed {
    border-image: url(:/images/spindown_pressed.png) 1;
}

QSpinBox::down-arrow {
    
	color: rgb(217, 217, 217);
    width: 7px;
    height: 7px;
}

QSpinBox::down-arrow:disabled,
QSpinBox::down-arrow:off { /* off state when value in min */
   image: url(:/images/down_arrow_disabled.png);
}
</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Cd">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>353</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Plotter">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>353</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_TapeDrive">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>353</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startSimulationButton</sender>
   <signal>clicked()</signal>
   <receiver>StartDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>StartDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartDialog</sender>
   <signal>rejected()</signal>
   <receiver>StartDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
